unlock-app/src/__tests__/components/helpers/Balance.test.js

```diff
@@ -5,17 +5,27 @@ import { Balance } from '../../../components/helpers/Balance'
 
 describe('Balance Component', () => {
   const unit = 'szabo'
+  const conversion = { USD: 195.99 }
 
   describe('when the balance is 0 Eth', () => {
     const amount = '0'
 
-    const wrapper = render(<Balance
-      amount={amount}
-      unit={unit}
-    />)
+    it('no conversion data available', () => {
+      const wrapper = render(<Balance
+        amount={amount}
+        unit={unit}
+        conversion={{ USD: undefined }}
+      />)
 
-    it('shows the value of 三 0', () => {
-      expect(wrapper.text()).toEqual('三 0')
+      expect(wrapper.text()).toEqual('0---')
+    })
+    it('USD conversion data available', () => {
+      const wrapper = render(<Balance
+        amount={amount}
+        unit={unit}
+        conversion={conversion}
+      />)
+      expect(wrapper.text()).toEqual('00')
```

Comment:
I wonder if there is no better way than testing the `text()` value here. This worked fine when using just Eth but here, this is probably not good enough.

Reply:
this is part of why I recommend using rtl for testing, it is easier to wiggle through the DOM, since all the search facilities are literally querySelector-based, but I can see if enzyme has a better way.

This brings up another issue with testing. When using styled-components, how do you search for a component? It may be possible to search for the component itself using `find()` I'll try that out. In other projects, I have preferred using css modules, because all the CSS is built statically, and so it's much easier to search for a component. However, this is a conversation for a different issue. :) For now, I'll find a solution that works with enzyme

Reply:
The most recent commit resolves this issue. Unfortunately, enzyme was incapable of resolving it (believe me, I tried, spent about 20 minutes with the manual, testing different things, and nada). After switching the test to react-testing-library, I had it resolved in less than 10 seconds. I was forced to move both the Balance and CreatorAccount tests over to rtl in order to test the new behavior.

Reply:
hooo sorry about that :( 
I just opened #467 for this.